PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Pull Request Template
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected, or that adds new dependencies)
+- [ ] This change requires a documentation update
```

Comment:
Just "documentation change" seems appropiate, we have those quite often

Reply:
Just wanted to comment one line here.
we should still have multiple things to tick.
Tho a checkbox that it's just a docs update would be a viable alternative

Reply:
yes